spec/requests/admin/site_settings_controller_spec.rb

```diff
@@ -125,6 +125,18 @@
 
           expect(CategoryUser.where(category_id: category_ids.first, notification_level: watching).count).to eq(0)
           expect(CategoryUser.where(category_id: category_ids.last, notification_level: watching).count).to eq(User.real.where(staged: false).count - 1)
+
+          topic = Fabricate(:topic, category_id: category_ids.last)
+          topic_user1 = Fabricate(:topic_user, topic: topic, notification_level: TopicUser.notification_levels[:watching], notifications_reason_id: TopicUser.notification_reasons[:auto_watch_category])
+          topic_user2 = Fabricate(:topic_user, topic: topic, notification_level: TopicUser.notification_levels[:watching], notifications_reason_id: TopicUser.notification_reasons[:user_changed])
+
+          put "/admin/site_settings/default_categories_watching.json", params: {
```

Comment:
Generally, I feel that it is a good practice to assert for the response of a request so that if the request does fail, it is easier to debug why.